.evergreen/config.yml

```diff
@@ -314,10 +285,10 @@ functions:
   "upload test results":
     - command: attach.results
       params:
-        file_location: "${PROJECT_DIRECTORY}/test-results.json"
+        file_location: "test-results.json"
     - command: attach.results
       params:
-        file_location: "${DRIVERS_TOOLS}/results.json"
```

Comment:
I don't think this will work, these are two different directories these files are in